OutOfSchool/OutOfSchool.ExternalFileStore.Impl/OutOfSchool.ExternalFileStore.Impl.csproj

```diff
@@ -23,7 +23,9 @@
 
     <ItemGroup>
         <PackageReference Include="Google.Cloud.Storage.V1" />
+        <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" />
```

Comment:
A place for background jobs is background jobs project. No need for abstraction to be added here